### ISSUE TITLE
[Feature Request] thumbnail navigation without pagination

### ISSUE THREAD

**Issue by kanjieater** [gotson/komga#657]
### Is your feature request related to a problem? Please describe.
When I'm setting up Komga to have my read status that other apps have, I want to jump to the exact page, which often is out of the range of pagination that komga displays.

### Describe the solution you'd like
Pagination doesn't seem like an ideal fit for this feature, but I also understand not wanting to have to download all the thumbnails for the archive. It would still be an ideal user experience to use a slider or other means to let you quickly skip to the desired page, even if pages are only fetched in "paginated" batches behind the scenes.

### Describe alternatives you've considered or other apps that can do what you want
Apps like ComicDisplayEx allow you to use a sliders to jump to a specific page.
![Screenshot_20210903-135622_CDisplayEx](https://user-images.githubusercontent.com/32607317/132053852-cb88ef41-3ac7-4e15-affc-d12d03cd2086.jpg)

![Screenshot_20210903-135925_CDisplayEx](https://user-images.githubusercontent.com/32607317/132054170-345bd1d9-2498-4391-9841-e5d1997f42e2.jpg)

**Comment by gotson** [comment]
The description is incorrect because that behavior is exactly the current behavior. Clicking a thumbnail takes you to that page. Can you rephrase what is the problem exactly? I don't understand what's the link with read status tbh.

**Comment by kanjieater** [comment]
The read status part: 
I want to jump to page around the middle of a series I started reading in another app before using komga. 

The issue with the interface today is the thumbnails are paginated instead of using a slider. 
![image](https://user-images.githubusercontent.com/32607317/132115732-42365f63-b4e7-45c4-b178-854dd7b7e377.png)

I do agree that it's nice that there's a slider for the "no thumbnail" navigation at the bottom.

**Comment by gotson** [comment]
If the problem is purely related to jumping to a specific page number then you probably don't need the thumbnail view, no? The slider should get you there pretty easily. It also seems to be a one time thing, right?

**Comment by kanjieater** [comment]
> If the problem is purely related to jumping to a specific page number then you probably don't need the thumbnail view, no? The slider should get you there pretty easily. It also seems to be a one time thing, right?

I was trying to give that as one solid use case, but in general I think it would be easier to use the thumbnail navigation with a slider like CDispalyEx as pictured for navigating quickly & visually. If you want to use the thumbnail navigation to find something quickly you have to guess by pagination number instead of page number. A slider could fix that.
